### PR TITLE
Feature/level1 entity mapping

### PR DESCRIPTION
**Pull request by Pontus-Wikstrom**
Made it possible to add many entities to one level.

### CODE REVIEW

**Comment by filipkauffeldt** [comment]
Would it be possible to have a map with less obstacles that looks better?

**Review by Oskmey — APPROVED** [review]
Looks good very nice!!

File: crocket/src/main/java/com/crocket/model/Level1.java [hunk 1]
```diff
@@ -33,17 +41,42 @@ public class Level1 extends Level {
             { GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, SAND, SAND, SAND, SAND, SAND, SAND, SAND, SAND, SAND, SAND, SAND, GRASS, GRASS, GRASS },
             { GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, GRASS, SAND, SAND, SAND, SAND, SAND,SAND, SAND, SAND, SAND, SAND, SAND, GRASS, GRASS, GRASS, GRASS} };
 
+    private static Stone[] stoneList = {
+            new Stone(STONEWIDTH, STONEHEIGHT, 970, 570),
+            new Stone(STONEWIDTH, STONEHEIGHT, 1070, 500),
+            new Stone(STONEWIDTH, STONEHEIGHT, 900, 400),
+            new Stone(STONEWIDTH, STONEHEIGHT, 1070, 200),
+            new Stone(STONEWIDTH, STONEHEIGHT,970, 0),
```
Comment (Dexetivia) [inline crocket/src/main/java/com/crocket/model/Level1.java:49]: Missing blankspace

**Review by Dexetivia — APPROVED** [review]

File: crocket/src/main/java/com/crocket/model/Level1.java (hunk 1 above)
Comment (Pontus-Wikstrom) [reply crocket/src/main/java/com/crocket/model/Level1.java]: Good catch! Will do.

**Pull request merged**